.internal-ci/test/fs-integration/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
```

Comment:
Is there a way to keep this file current with the one in the main FS directory?

Reply:
Check out the latest changes and the README I included as part of the  `.internal-ci/test/fs-integration` directory for setting up the environment.  We should move the work that's in `ineg-tests` in to this directory.  We can use the same `poetry` project for multiple test scripts if we want.

As full-service is not a python project, I don't think the testing frameworks dependencies or setups should be part of the main repo structure. 

In the longer term, I think we should move/rename the `cli` directory and possibly clean up or delete the `python-utils`.  Some of the tools that get reused here should get broken out into a separate repo and properly versioned instead of copied around to various projects.

Reply:
Check out the latest changes and the README I included as part of the  `.internal-ci/test/fs-integration` directory for setting up the environment.  We should move the work that's in `ineg-tests` in to this directory.  We can use the same `poetry` project for multiple test scripts if we want.

As full-service is not a python project, I don't think the testing frameworks dependencies or setups should be part of the main repo structure. 

In the longer term, I think we should move/rename the `cli` directory and possibly clean up or delete the `python-utils`.  Some of the tools that get reused here should get broken out into a separate repo and properly versioned instead of copied around to various projects.